### PR TITLE
2065: enabled logging for postgres

### PR DESCRIPTION
**Pull request by erezmus**
Enables logging for postgres

### CODE REVIEW

File: docker-compose.override.yml [hunk 1]
```diff
@@ -20,6 +20,9 @@ services:
     volumes:
       - postgres-volume:/var/lib/postgresql/data
       - ./scripts/test.sql:/docker-entrypoint-initdb.d/test.sql
+      - ./build/logs/postgres:/var/lib/postgresql/data/logs
```
Comment (diversemix) [inline docker-compose.override.yml]: I'm assuming if the `logs` folder does not exist on the developer's machine then it will get created?

**Review by diversemix — APPROVED** [review]

**Review by will-byrne — APPROVED** [review]

File: Jenkinsfile [hunk 2]
```diff
@@ -32,6 +32,7 @@ elifePipeline {
                   } finally {
                       sh "sudo docker ps -a"
                       sh "sudo sh -c \"docker logs elife-xpub_postgres_1 > build/logs/unit-postgres-output.txt\""
```
Comment (giorgiosironi) [inline Jenkinsfile]: this line can also be problematic as `build/logs/unit-postgres-output.txt` will be created with `root` ownership?

File: Jenkinsfile [hunk 3]
```diff
@@ -32,6 +32,7 @@ elifePipeline {
                   } finally {
                       sh "sudo docker ps -a"
```
Comment (giorgiosironi) [inline Jenkinsfile]: remove `sudo` from here

**Comment by giorgiosironi** [comment]
Artifacts available at https://alfred.elifesciences.org/blue/organizations/jenkins/pull-requests-projects%2Felife-xpub/detail/PR-2066/25/artifacts
I suggest not to use `build/logs` as a target of a copy because it's [mounted in the container](https://github.com/elifesciences/elife-xpub/blob/develop/docker-compose.ci.yml#L49).

**Review by Cooryd — APPROVED** [review]

**Comment by diversemix** [comment]
Can we merge this now?

**Comment by erezmus** [comment]
@diversemix we can, but was wondering if we could squash the commits?

**Comment by diversemix** [comment]
I always have the button "Squash and merge" :)

**Pull request merged**